tasks/packages.yml

```diff
@@ -17,8 +17,30 @@
 # limitations under the License.
 #
 
+- include: packages_ubuntu.yml
+  when: ansible_distribution == 'Ubuntu'
```

Comment:
So, should we check `ansible_distribution` instead of `ansible_os_family` ? I think the latter is more appropriate because it refers to the packaging system (both redhat and centos are under 'RedHat' family,  both debian and ubuntu are apt, etc) but if it sounds confusing, as there's no maintaned Openstack for Debian, then it's ok.